README.md

```diff
@@ -125,7 +100,9 @@ PNxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx  +1209337XXXX  DAVID'S TWILIO CONTACT
 
 Many `list` commands will allow you to specify a `--properties` option to specify which columns you'd like to display. For example, to display only the Phone Number and SMS Url columns, you would pass `--properties "phoneNumber, smsUrl"`.
 
-Note that currently, the column names must match the JSON property names in the Twilio API.
+Note that the default list of properties varies by command and is subject to change with each release.
+
+Also note that currently, the column names must match the JSON property names in the Twilio API.
```

Comment:
"Also note that currently, the column names must match the JSON property names in the Twilio API." -> "Also note that the column names must match the JSON property names in the Twilio API."

Reply:
I just moved this line, but I'm fine with this change.